ait-extension/src/js/components/VideoDetailsModal.tsx

```
@@ -0,0 +1,801 @@
+                  <p className="no-thumbnails">No thumbnails available</p>
+                )}
+              </div>
+            )}
+
+            {activeTab === 'camera' && (
+              <div className="tab-pane">
+                <div className="metadata-rows">
+                  <div className="metadata-row">
+                    <div className="metadata-row-label"><Camera size={16} /> Make</div>
+                    <div className="metadata-row-value">
+                      {displayClip.camera_details && displayClip.camera_details.make 
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)